.github/workflows/test_e2eshark.yml

```diff
@@ -0,0 +1,100 @@
+# Copyright 2024 Advanced Micro Devices
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: E2ESHARK Test Suite
+on:
+  workflow_dispatch:
+  schedule:
+  # Runs at 1:00 AM UTC, which is 5:00 AM PST (UTC-8)
+   - cron: '0 13 * * *'
+
+jobs:
+  turbine:
+    runs-on: nodai-amdgpu-mi250-x86-64
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          repository: nod-ai/SHARK-Turbine
+          ref: torch_2.1_support
+          path: turbine
+
+  e2eshark:
+    needs: [turbine]
+    runs-on: nodai-amdgpu-mi250-x86-64
+    env:
+      E2E_VENV_DIR: ${{ github.workspace }}/test-suite_venv
+      TURBINE_VENV_DIR: ${{ github.workspace }}/turbine_venv
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          repository: nod-ai/SHARK-TestSuite
+          path: test-suite
+
+      - name: "Setup e2eshark python venv"
+        run: |
+          python3.11 -m venv ${E2E_VENV_DIR}
+          pip install --upgrade pip
+          pip install -r ./e2eshark/requirements.txt
+        working-directory: ./test-suite
+
+      - name: Run Onnx Mode
```

Comment:
https://github.com/nod-ai/SHARK-TestSuite/actions/runs/8895268824/job/24425273404#step:4:577
browsing through the logs, gpt2-xl looks pretty slow (sorta expected given the name):
![image](https://github.com/nod-ai/SHARK-TestSuite/assets/4010439/9e0cf964-5bce-4c7f-bd6d-5e920929605b)

Reply:
Yeah, we can leave it for now given it's a nightly